Tetrahedral_remeshing/examples/Tetrahedral_remeshing/tetrahedral_remeshing_with_features_and_sizing.cpp

```diff
@@ -0,0 +1,121 @@
+#define CGAL_DUMP_REMESHING_STEPS
+#define CGAL_TETRAHEDRAL_REMESHING_VERBOSE
+#define CGAL_TETRAHEDRAL_REMESHING_DEBUG
+
+// experiments on sizing
```

Comment:
Is this intentionally in an example or a leftover from your experiments @janetournois ?